crates/next-core/src/next_client/context.rs

```diff
@@ -238,7 +238,7 @@ pub async fn get_client_runtime_entries(
             .as_request();
 
     let mut runtime_entries = vec![RuntimeEntry::Ecmascript(
-        ProcessEnvAssetVc::new(project_root, env_for_js(env, true)).into(),
+        ProcessEnvAssetVc::new(project_root, env_for_js(env, true, None)).into(),
```

Comment:
Why isn't a next config used here?

Reply:
Pass `NextConfigValueVc` into `env_for_js` here need to pass `NextConfigValueVc` into `fn app_context` and the `NextConfigVaueVc` is built from it.
It's a temporary solution, I'll build a separate `AssetContext` for `fn load_next_config` later